phalcon/factoryinterface.zep

```diff
@@ -25,5 +25,5 @@ interface FactoryInterface
 	/**
 	 * @param \Phalcon\Config|array config
 	 */
-	public static function load(var config) -> object;
+	public static function load(var config);
```

Comment:
Usually this method used as factory method. So it should return the result of their work. Isn't?

Reply:
i need some help here, I don't know. This method returns different object every time. And php was complaining about this return type "object". I do not know ...

Reply:
Just leave as `var`